website/static/js/comment.js

```diff
@@ -122,22 +123,26 @@ BaseComment.prototype.setupToolTips = function(elm) {
 
 BaseComment.prototype.fetch = function() {
     var self = this;
+    var setUnread = getTargetType(self) !== 'comments' && !osfHelpers.urlParams().view_only && self.author.id !== '';
```

Comment:
Since `getTargetType` relies on `this`/`self`, it should be a method rather than a function.